packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -467,6 +491,20 @@ class CupertinoSlidingSegmentedControl<T extends Object> extends StatefulWidget
   /// Defaults to `EdgeInsets.symmetric(vertical: 2, horizontal: 3)`.
   final EdgeInsetsGeometry padding;
 
+  /// Determines whether segments in the segmented control show selected state.
```

Comment:
This should explain what showing selected state is, although it might be better described as temporary feedback when a segment is selected.